openid-4-verifiable-presentations-1_0.md

```diff
@@ -1492,9 +1492,139 @@ issuers in Self-Sovereign Identity ecosystems using TRAIN</title>
         </front>
 </reference>
 
-# Usage with Various Credential Formats {#alternative_credential_formats}
+<reference anchor="w3c.digital_credentials_api" target="https://wicg.github.io/digital-credentials/">
+        <front>
+          <title>Digital Credentials API</title>
+		  <author fullname="Marcos Caceres">
+            <organization>Apple Inc.</organization>
+          </author>
+          <author fullname="Sam Goto">
+            <organization>Google</organization>
+          </author>
+        </front>
+</reference>
+
+# OpenID4VP profile for the W3C Digital Credentials API
```

Comment:
A credential format may need to specify the cryptographic details that are specific for how it is used in conjunction with the browser api, e.g. device/key binding, or how the nonce parameter is used. Therefore, we need to add a section to this PR that defines these normative details.

Reply:
that applies to an entire spec and is not specific to browser API. we already have credential format profiles, so nothing special is needed. please read appendix A

Reply:
I don’t think this issue is resolved. We need to specify these details somewhere. For example in the case where mdoc is used, it needs to be specified what the contents is of the SessionTranscript structure. This will be specific for using the browser api and will (likely) be different when using other OpenID4VP flows.
I don’t think it matters too much whether this is specified in Annex A or the Browser API section, but Annex A currently says ‘examples’ while these are normative requirements.

Reply:
Is there anything beyond the SessionTranscript that needs to be specified, and I think for SessionTranscript it's only what to put in 'responseUri'? (e.g. I'm currently struggling to think of anything extra that would need to be specified for SD-JWT nor can I think of anything extra we need to say about nonce beyond what's already said).

I'm not sure if we would plan to specify this in VP or leave it to ISO. The current draft completely defers to ISO for mDL specifics.

Appendix A has already been renamed to remove 'examples' as it already had some normative requirements in it; I think this would likely be the place to put it if we do want to say something mDL/mDoc specific in the VP spec. FYI This version of the spec is auto updated when 'main' in this git repo is updated: https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#name-usage-with-various-credenti

Could you open an issue for the above please Martjin? (We're keen to get this initial PR merged as it's approaching 3 months old now and we have a number of issues already open that we can't do PRs for until this is merged.)

Reply:
Discussed on today's WG call. There was a consensus around creating some kind of initial profile for how mdoc/mDL could work over the browser API and Oliver will open an issue.